src/shared/molecules/DataExplorerGraphFlowMolecules/styles.less

```diff
@@ -0,0 +1,180 @@
+@import '../../lib.less';
+
+.navigation-stack-item {
+  display: grid;
+  grid-template-rows: repeat(4, max-content);
+  gap: 10px;
+  align-items: start;
+  justify-items: center;
+  justify-content: center;
+  align-content: start;
+  border-right: 1px solid #bfbfbf;
+  padding: 3px 5px;
+  height: 100%;
+  min-height: 0;
+  background-color: #f5f5f5;
+
+  &.more {
+    background-color: white;
+  }
+
+  &.no-more {
+    display: none;
+    transition: display 0.2s cubic-bezier(0.075, 0.82, 0.165, 1);
+  }
+
+  &.highlight {
+    border: 2px solid #377af5;
+    transition: border-color 0s ease-out infinite;
+  }
+
+  &.shrinkable {
+    display: none;
+  }
+
+  .icon {
+    margin-top: 5px !important;
+    padding: 4px !important;
+  }
+
+  .title {
+    font-family: 'Titillium Web';
```

Comment:
I might be wrong, so do double check, but I think there are default fonts set in the `lib.less` file for everything that also take care of graceful fallbacks. Given that, I don't think we should override the font-family here (or in other places too).